src/MongoDB.Driver.Core/Core/Servers/Server.cs

```diff
@@ -810,6 +810,40 @@ public void Dispose()
                 return ExecuteProtocolAsync(protocol, cancellationToken);
             }
 
+            public CursorBatch<TDocument> Query<TDocument>(
+                CollectionNamespace collectionNamespace,
+                BsonDocument query,
+                BsonDocument fields,
+                IElementNameValidator queryValidator,
+                int skip,
+                int batchSize,
+                bool slaveOk,
+                bool partialOk,
+                bool noCursorTimeout,
+                bool tailableCursor,
+                bool awaitData,
+                IBsonSerializer<TDocument> serializer,
+                MessageEncoderSettings messageEncoderSettings,
+                CancellationToken cancellationToken)
+            {
+                return Query(
+                    collectionNamespace,
+                    query,
+                    fields,
+                    queryValidator,
+                    skip,
+                    batchSize,
+                    slaveOk,
+                    partialOk,
+                    noCursorTimeout,
+                    oplogReplay: false,
+                    tailableCursor,
+                    awaitData,
+                    serializer,
+                    messageEncoderSettings,
+                    cancellationToken);
+            }
+
             public CursorBatch<TDocument> Query<TDocument>(
```

Comment:
Can we deprecate this method as we did here: https://github.com/rstam/mongo-csharp-driver/blob/bc06c66270fd721514c41370417f1bbea298b508/src/MongoDB.Driver.Core/Core/Servers/Server.cs#L450

Reply:
I hadn't thought it was necessary because this is a private class... but I don't see any reason not to.

Reply:
Done